bot/bot.go

```diff
@@ -111,8 +111,11 @@ func (b *Bot) Init(rtm *slack.RTM) error {
 
 	params := slack.PostMessageParameters{AsUser: true}
 	_, _, err = b.slackBotAPI.PostMessage(b.users["dlsniper"], fmt.Sprintf(`Deployed version: %s`, b.version), params)
+	if err != nil {
+		b.logf("Failed to send deploy version message.")
+	}
 
-	return err
+	return nil
```

Comment:
Was due to the fact that it got pretty hard to test outside the real Slack channel :)